Documentation/Microsoft.Support.md

```diff
@@ -0,0 +1,12 @@
+# Microsoft Support
+
+This document details Microsoft Support pollicy for CoreWCF.
+
+Runtime Version | Support dependency duration |
+--- | --- |
+.NET Framework 4.x | The specified version of .NET Framework and ASP.NET Core 2.1 |
```

Comment:
Is it worth making ".NET Framework" and "ASP.NET Core 2.1" link to https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework and https://dotnet.microsoft.com/en-us/platform/support/policy/aspnetcore-2.1 respectively?

Reply:
make sense